web/experimental/styles.scss

```diff
@@ -196,6 +197,8 @@ body {
   z-index: 10;
   font-size: 18px;
   color: $mdc-theme-primary;
+  margin-right: 1em;
```

Comment:
Almost all the other margin/padding measurements in here are in raw pixels. Is this one in ems for a particular reason?

Reply:
This is a holdover from the old playground styles. changed to 8px.